src/resources/2024/ChestMimic.ts

```diff
@@ -136,17 +136,14 @@ export function differentiate(
 /**
  * Check how many of a monster is available to differentiate into
  *
- * @param monster The monster to differentiate your egg into; may behave weirdly with name collisions
+ * @param monster The monster to differentiate your egg into
  * @returns How many of a Monster we can differentiate
  */
 export function differentiableQuantity(monster: Monster): number {
   if (!have_($item`mimic egg`)) return 0;
-  const regex = new RegExp(
-    `<!-- monsterid: ${monster.id}-->(?:\\s\\((\\d+)\\))?`
-  );
+  const regex = new RegExp(`<!-- monsterid:${monster.id} -->\\s\\((\\d+)\\)`);
   const page = examine($item`mimic egg`);
   const match = page.match(regex);
-  if (!match) return 0;
-  if (!match[1]) return 1;
+  if (!match?.[1]) return 0;
   return clamp(Number(match[1]), 1, availableAmount($item`mimic egg`));
```

Comment:
```suggestion
  return Math.min(Number(match[1]) || 0, itemAmount($item`mimic egg`));
```
Should be itemAmount for that one guy (HolderOfSecrets)

Reply:
Oops nevermind this won't work how I thought. Can't do `undefined[1]`